server/src/main/java/io/littlehorse/server/streams/storeinternals/ReadOnlyGetableManager.java

```diff
@@ -84,6 +84,14 @@ protected <U extends Message, T extends CoreGetable<U>> List<GetableToStore<U, T
         return store.range(start, end, cls);
     }
 
+    public <T extends Storeable<?>> LHKeyValueIterator<T> scanStoreables(String start, String end, Class<T> cls) {
+        return store.range(start, end, cls);
+    }
+
+    public <U extends Message, T extends Storeable<U>> T get(String storeKey, Class<T> cls) {
```

Comment:
This is somewhat of a leaky abstraction, is it needed?

Reply:
I would prefer that it is a `public ScheduledTaskModel get(TaskRunId taskRunId)`. I don't think we should be exposing arbitrary `Storeable`s in the `GetableManager`, since the `GetableManager` manages `Getable`s (not `Storeable`s)